accepted/0000-hub-online-synchronization.md

```diff
@@ -122,18 +122,34 @@ During development phase we should consider if we can join both namespaces in on
 
 
 ### Alternative 2:
-We can follow a similar approach to what we do to activate a proxy.
-On the Hub we create an API which is called authenticated and can take additional data like credentials for the peripheral server.
-This API create the configuration on the Hub side and also handle the creation of a system which represent an Uyuni Peripheral Server.
 
-On the peripheral side we call the API on the Hub and fill out the required configuration data needed locally on the peripheral server (e.g. auto generated mirror credential password).
-We will also change the `scc_url` configuration on the Peripheral to point to the Hub Server.
+We need to establish a secure connection between Hub and Peripheral Server.
```

Comment:
@rjmateus @mcalmer are we going to drop the existing way of Hub and peripheral servers way of work? What would be the hub_xmlrpc_api role, will that be gone too? 

What path are we providing for the users who are working with the current approach.

Reply:
hub_xmlrpc_api is for ISSv2. In worst case the user can stay in it.
I cannot say if this is compatible with ISSv3. I do not really know what this is doing.
@rjmateus might be able to answer this

Reply:
hub_xmlrpc_api will stay as is, since it's note dependent on the ISS version in use. This means it can work with any version of ISS, even the v3.
However, we may need to make some changes to one API that is providing the hub servers FQDN to the xml-rpc API, to instead of looking at the system entitlement it returns a list of configured peripherals

Reply:
And what about all different authentication methods https://documentation.suse.com/suma/5.0/en/suse-manager/specialized-guides/large-deployments/hub-auth.html ? 
To support them, we will leave it up to the user to create the appropriate API users, ensuring that those users can be utilized to correctly make the API calls later on?